docs/upgrading.md

```diff
@@ -5,6 +5,17 @@ Below we describe the additional manual steps required to upgrade between differ
 
 # Open Autonomy
 
+## `v0.16.1` to `v0.17.0`
+
+- The structure of the build folder when using Docker has been updated. 
+  The folder is now named `abci_build_{{index}}`, 
+  where `{{index}}` is an incrementing number corresponding to the total number of existing builds, 
```

Comment:
They are now like `abci_build_nki5`
```suggestion
  The folder is now named `abci_build_{{uuid}}`, 
  where `{{uuid}}` is a 4 character random hash., 
```

Reply:
Replaced with https://github.com/valory-xyz/open-autonomy/pull/2268#discussion_r1803012890 as this was marked as outdated after force pushing.